frontend/src/utils/__tests__/json-parser.test.ts

```diff
@@ -131,3 +135,158 @@ it("handles NaN values in TSV", () => {
 
   expect(jsonToTSV([{ a: Number.NaN, b: 1 }], locale)).toEqual("a\tb\nNaN\t1");
 });
+
+it("can convert json to markdown - basic table", () => {
+  expect(jsonToMarkdown([])).toMatchInlineSnapshot(`""`);
+
+  expect(jsonToMarkdown([{ a: 1, b: 2 }])).toMatchInlineSnapshot(`
+    "| a | b |
+    |---|---|
+    | 1 | 2 |"
+  `);
+
+  expect(
+    jsonToMarkdown([
+      { a: 1, b: 2 },
+      { a: 3, b: 4 },
+    ]),
+  ).toMatchInlineSnapshot(`
+    "| a | b |
+    |---|---|
+    | 1 | 2 |
+    | 3 | 4 |"
+  `);
+
+  expect(
+    jsonToMarkdown([
+      { name: "Alice", age: 30 },
+      { name: "Bob", age: 25 },
+    ]),
+  ).toMatchInlineSnapshot(`
+    "| name | age |
+    |---|---|
+    | Alice | 30 |
+    | Bob | 25 |"
+  `);
+});
+
+it("can convert json to markdown - with URLs", () => {
+  expect(
+    jsonToMarkdown([
+      { name: "Google", url: "https://google.com" },
+      { name: "GitHub", url: "https://github.com" },
+    ]),
+  ).toMatchInlineSnapshot(`
+    "| name | url |
+    |---|---|
+    | Google | [https://google.com](https://google.com) |
```

Comment:
Maybe want to include a test of the expected behavior when there is a markdown link already? I would assume we generally wouldn't expect those inputs, but it would result in a nested link.